From: Keith Packard <keithp@keithp.com>
Date: Mon, 7 Sep 2020 16:18:53 -0700
Subject: Update gemspec dependencies to match debian packages

I've tested with these versions and the test suites results match the
current debian package, although there are spurious errors, e.g. 12 !=
12.0.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 asciidoctor-pdf.gemspec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/asciidoctor-pdf.gemspec b/asciidoctor-pdf.gemspec
index e34ad88..3495695 100644
--- a/asciidoctor-pdf.gemspec
+++ b/asciidoctor-pdf.gemspec
@@ -34,9 +34,9 @@ Gem::Specification.new do |s|
   #s.test_files = files.grep %r/^(?:test|spec|feature)\/.*$/
 
   s.add_runtime_dependency 'asciidoctor', '>= 1.5.3', '< 3.0.0'
-  s.add_runtime_dependency 'prawn', '~> 2.2.0'
+  s.add_runtime_dependency 'prawn', '>= 2.2.0'
   # NOTE ttfunk 1.6 is not compatible with Ruby 2.3, so lock version to 1.5 for consistency
-  s.add_runtime_dependency 'ttfunk', ['~> 1.5.0'], ['>= 1.5.1']
+  s.add_runtime_dependency 'ttfunk', ['>= 1.5.1']
   # NOTE must use prawn-table from master branch (defined in Gemfile) for full functionality
   s.add_runtime_dependency 'prawn-table', '~> 0.2.0'
   # s.add_runtime_dependency 'prawn-templates', '~> 0.1.0'
